applications/st2110_test/README.md

```
@@ -0,0 +1,278 @@
+
+- NVIDIA Thor AGX (or compatible system)
+- NVIDIA ConnectX-6 or later NIC (or Thor AGX MGBE)
+- ST 2110-20 video source streaming to the network
+
+### Software
+
+- ST2110SourceOp operator built (see `../../operators/st2110_source/BUILD.md`)
+- DPDK configured and NIC bound to vfio-pci driver
+- ST 2110 video source streaming to multicast 239.0.0.1:5004
+
+## Quick Start
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)